includes/payment-methods/class-wc-stripe-upe-payment-method-cc.php

```diff
@@ -22,7 +22,7 @@ class WC_Stripe_UPE_Payment_Method_CC extends WC_Stripe_UPE_Payment_Method {
 	public function __construct() {
 		parent::__construct();
 		$this->stripe_id   = self::STRIPE_ID;
-		$this->title       = 'Credit card / debit card';
+		$this->title       = __( 'Pay with credit card / debit card', 'woocommerce-gateway-stripe' );
```

Comment:
In the issue, "credit card" is not supposed to contain "Pay with". Is it intentional?

Reply:
I'm pretty sure what @diegocurbelo put was the titles/labels as he saw them in the code when writing the issue, including the inconsistencies (i.e., the missing "Pay with" and Sepa vs SEPA.